macros/core/PGbasicmacros.pl

```diff
@@ -2093,6 +2093,7 @@ sub old_safe_ev {
 
 sub FEQ {    # Format EQuations
 	my $in = shift;
+	return '' if ( ! defined($in) ); # No need to do all the string replacements in this case
```

Comment:
This should be
```suggestion
	return '' if !defined $in;    # No need to do all the string replacements in this case

```
The spacing is incorrect for perltidy, and the parentheses aren't needed.  The new empty line is needed to ensure that the comment on the next line is not indented to match the comment you added.  perltidy will do that.  Generally comments at the end of a line aren't the best practice and shouldn't be used except for very short identifier comments in lists.